build-tools-internal/src/main/groovy/elasticsearch.build-scan.gradle

```diff
@@ -118,6 +121,13 @@ buildScan {
         }
       }
 
+      def uptime = ManagementFactory.getRuntimeMXBean().getUptime() / 1000;
+      def metricsStartTime = LocalDateTime.now().minusSeconds(uptime.longValue()).minusMinutes(15).toString()
```

Comment:
I assume Kibana uses local user time, not UTC?

Reply:
I'm including a `Z` on the end of the timestamp in the URL below, which forces the timestamp to be parsed using UTC (but still presented using user time)